src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -3555,6 +3730,21 @@ public bool Reachable
                     return _knownNullState.Capacity > 0;
                 }
             }
+
+#if DEBUG
+            public override string ToString()
+            {
+                var pooledBuilder = PooledStringBuilder.GetInstance();
+                var builder = pooledBuilder.Builder;
+                builder.Append(" ");
+                for (int i = this.Capacity - 1; i >= 0; i--)
+                {
+                    builder.Append(_knownNullState[i] ? (_notNull[i] ? "! " : "? ") : ". ");
+                }
+
+                return pooledBuilder.ToStringAndFree();
+            }
```

Comment:
Consider removing the space between each `!?.` so the string representation aligns with the display of the two `BitVector` instances in the debugger.